src/components/eventList/eventList.js

```diff
@@ -1,34 +1,101 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Event from '../event/event';
+import uniqid from 'uniqid';
 
+import Event from '../event/event';
+import Loading from '../loading/loading';
+import RequestField from '../requestField/requestField';
+import {
+  SPRAVA_API_MAIN_URL, CORS_URL, SPRAVA_API_URL, ITEMS_STATUS_PARAMS,
+} from '../../constants/constants';
+import FESTIVAL_DAYS from '../../app-config/app-config';
 import './eventList.scss';
 
-const EventList = ({ events }) => (
-  <div className="events-section__event-list" id="event-list">
-    {events.map(e => (
-      <Event
-        key={e.id}
-        id={e.id}
-        eventName={e.eventName}
-        location={e.location}
-        time={e.time}
-        description={e.description}
-        isEventSave={Boolean(e.checked)}
-      />
-    ))}
-  </div>
-);
+class EventList extends React.Component {
+  componentDidMount() {
+    const { fetchData } = this.props;
+    fetchData(`${CORS_URL}/${SPRAVA_API_MAIN_URL}/${SPRAVA_API_URL.music}`);
+  }
+
+  render() {
+    const {
+      events,
+      fetchStatus,
+    } = this.props;
+    if (fetchStatus === ITEMS_STATUS_PARAMS.ITEMS_HAS_ERRORED) {
+      return <RequestField />;
+    }
+
+    if (fetchStatus === ITEMS_STATUS_PARAMS.ITEMS_IS_LOADING) {
+      return <Loading />;
+    }
+
+    return (
+      <div className="events-section__event-list-container" id="event-list">
+        <div
+          className="events-section__first-day-events"
+          style={(Object.keys(events)
+            .filter(e => events[e].date === `${FESTIVAL_DAYS.FIRST_DAY}`).length > 0)
```

Comment:
could we move that filter to separate function ? it is too long and hard to read among the markup